### PR TITLE
Toponaming:  bring in SketchObject::InternalShape

### PR DESCRIPTION
**Pull request by bgbsww**
Sixth in series of incremental code transfers to solve the save/restore errors with objects with element maps..

### CODE REVIEW

File: src/Mod/Sketcher/App/SketchObject.cpp [hunk 1]
```diff
@@ -337,9 +360,78 @@ void SketchObject::buildShape()
...
+            result = result.makeElementFace(result.getSubTopoShapes(TopAbs_WIRE),
+                    /*op*/"",
+                    /*maker*/"Part::FaceMakerRing",
+                    /*pln*/nullptr
+            );
+        }
+        Part::TopoShape openWires(getID(), getDocument()->getStringHasher());
+        joiner.getOpenWires(openWires, "SKF");
+        if (openWires.isNull())
+            return result;
```
Comment (chennes) [inline src/Mod/Sketcher/App/SketchObject.cpp:423]: Even if it's null?

Comment (bgbsww) [reply src/Mod/Sketcher/App/SketchObject.cpp]: Yep.  Added comments though.

**Comment by Roy-043** [comment]
I am getting an error that I think is related to this PR.
19:33:14  <Sketch> SketchObject.cpp(469): Failed to make face for sketch: Class 'Part::FaceMakerRing' not found.

And, indeed, FaceMakerRing does not occur anywhere in .../Mod/Part/
Maybe FaceMakerBullseye is intended?

Q: is it true that this PR enables the selection of subelements from a sketch for a PartDesign Pad operation?
Forum topic: https://forum.freecad.org/viewtopic.php?t=89532

**Comment by bgbsww** [comment]
Yeah, that slipped through.  FaceMakerRing is defined in LS3, so this needs
to be corrected.  Thanks!

A:  It appears to, accidentally.  That could equally get taken away
accidentally.

**Pull request merged**